tests/baselines/reference/truthinessCallExpressionCoercion.errors.txt

```diff
@@ -1,17 +1,17 @@
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(2,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(18,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(36,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(50,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(66,13): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(76,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-tests/cases/compiler/truthinessCallExpressionCoercion.ts(82,9): error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
+tests/cases/compiler/truthinessCallExpressionCoercion.ts(2,9): error TS2774: This condition will always return true since this function appears to always be defined. Did you mean to call it instead?

```

Comment:
👋 I think I first introduced this error message back when I contributed the first uncalled function check. I admit I didn't give it in-depth thought and it's kind of bothered me since. Since it's being adjusted, can I suggest one additional tweak?

```suggestion
tests/cases/compiler/truthinessCallExpressionCoercion.ts(2,9): error TS2774: This condition will always evaluate to true since this function appears to always be defined. Did you mean to call it instead?
```
Rationale being that `return` is a function-level concern, but these are almost always expressions.

Reply:
That also kind of bugged me - what about "This condition will always **be** true"?

Reply:
Even better!

Reply:
I don't like swapping "appears to always be" with "is". The wiggle room that the words give you isn't worth the extra reading, and the associated drop in number of people who will read to the end.

Just giving an incorrect error, regardless of wording, drops the compiler's credibility to zero, so you might as well be confident and clear in your wording.

Reply:
@sandersn we keep getting very annoying discussions about "will always be true because the types have no overlap" which is not really true.